.github/workflows/android.yaml

```diff
@@ -28,17 +24,20 @@ jobs:
     with:
       platform: android-arm64
       nightly: ${{ github.event.inputs.nightly }}
+      postsubmit: ${{ github.event_name == 'push' }}
```

Comment:
Any particular reason we need 'postsubmit' input parameter, why not just use github.event_name to check event?  The only reason I used 'nightly' input parameter in addition to github.event_name is to allow running nightly workflows via workflow_dispatch.

Reply:
When we want to run the tests manually, this is the only option we have to run on-device tests. I can replace this with the condition of course if we don't want to support on-device for manual triggering (debugging).

Reply:
When you say "manually" what do you mean exactly by that do you mean 'workflow_dispatch', if so it doesn't look like with this code on device test will run on 'workflow_dispatch'.  I am still failing to see why we need 'postsubmit' parameter if it looks at ${{ github.event_name == 'push' }}.  Keep in mind that postsubmit build can be either started by push event or via workflow_dispatch.

Reply:
Resolved. We are only restricting the on-device tests run for pre-submit